src/main/java/com/snowflake/kafka/connector/internal/telemetry/SnowflakeTelemetryServiceV1.java

```diff
@@ -140,6 +189,49 @@ private String getTaskID() {
     return taskID;
   }
 
+  /**
+   * Adds specific user provided connector properties to ObjectNode
+   *
+   * @param userProvidedConfig user provided key value pairs in a Map
+   * @param dataObjectNode Object node in which specific properties to add
+   */
+  protected void addUserConnectorPropertiesToDataNode(
+      final Map<String, String> userProvidedConfig, ObjectNode dataObjectNode) {
+    // maxTasks value isn't visible if the user leaves it at default. So 0
+    // means unknown.
```

Comment:
but do you prevent user from setting maxTasks to 0?

Reply:
Actually why not just use string? it will NULL if user doesn't set it

Reply:
there is actually no checks for tasks.max.

Reply:
It will change the values moving forward. i.e if nothing is given, it will parse it to null instead of 0.